assigner.py

```diff
@@ -38,23 +37,21 @@ def main():
 
     # Do things!
     try:
-        with open('assigner.yml') as fd:
+        with open(args.configfile) as fd:
             config = yaml.load(fd)
     except e:
-        logger.critical("Could not parse configuration file assigner.yml: {}".format(e))
+        logger.critical("Could not parse configuration file: {}".format(e))
         sys.exit(127)
     bcfg = config.get('bugzilla')
-    bapi = bugzilla.Bugzilla(url=bcfg.get('url'), api_key=bcfg.get('api_key'))
+    bapi = bugzilla.Bugzilla(url=bcfg.get('url'), api_key=os.environ.get('BUGZILLA_API_KEY'))
 
-    if args.rra:
-        autoassign_rras(bapi, bcfg.get('rra'), args.dry_run)
+    #autoassign(bapi, bcfg.get('rra'), args.dry_run)
```

Comment:
I have this disabled temporarily until the pre-existing process for auto-assignment is decommed.

I would like to start using this for VA auto-assignment as soon as it's ready.